oasislmf/model_execution/bin.py

```diff
@@ -151,6 +155,18 @@ def prepare_run_directory(
                     os.symlink(path, os.path.join(model_data_dst_fp, fn))
             except Exception:
                 shutil.copytree(model_data_fp, os.path.join(model_data_dst_fp, fn))
+
+        if user_data_dir and os.path.exists(user_data_dir):
```

Comment:
Log a warning or fail if the specified directory doesn't exist? Otherwise it might take us a while to figure out that there's a typo in the directory path.